readme.md

```diff
@@ -68,495 +35,38 @@ From the npm registry:
-| [`rate-limit-redis`](https://npmjs.com/package/rate-limit-redis)                             | A [Redis](http://redis.io/)-backed store, more suitable for large or demanding deployments.                                                   | Modern as of v3.0.0 |
-| [`rate-limit-memcached`](https://npmjs.org/package/rate-limit-memcached)                     | A [Memcached](https://memcached.org/)-backed store.                                                                                           | Modern as of v1.0.0 |
-| [`rate-limit-mongo`](https://www.npmjs.com/package/rate-limit-mongo)                         | A [MongoDB](https://www.mongodb.com/)-backed store.                                                                                           | Legacy              |
-| [`precise-memory-rate-limit`](https://www.npmjs.com/package/precise-memory-rate-limit)       | A memory store similar to the built-in one, except that it stores a distinct timestamp for each key.                                          | Modern as of v2.0.0 |
-| [`rate-limit-postgresql`](https://www.npmjs.com/package/@acpr/rate-limit-postgresql)         | A [PostgreSQL](https://www.postgresql.org/)-backed store.                                                                                     | Modern as of v1.1.0 |
-
-Take a look at
-[this guide](https://github.com/express-rate-limit/express-rate-limit/wiki/Creating-Your-Own-Store)
-if you wish to create your own store.
-
-## Request API
-
-A `req.rateLimit` property is added to all requests with the `limit`, `used`,
-and `remaining` number of requests and, if the store provides it, a `resetTime`
-Date object. These may be used in your application code to take additional
-actions or inform the user of their status.
-
-Note that `used` includes the current request, so it should always be > 0.
-
-The property name can be configured with the configuration option
-`requestPropertyName`.
-
-## Instance API
-
-### `resetKey(key)`
-
-Resets the rate limiting for a given key. An example use case is to allow users
-to complete a captcha to reset their rate limit, then call this function.
-
-### `getKey(key)`
-
-Retrieves the hit count and reset time from the store for a given key.
+A more in-depth documentation is available on-line at
+[express-rate-limit.mintlify.app](https://express-rate-limit.mintlify.app/overview).
+If you are offline, you can view the documentation sources directly - they are
+written in `mdx` and stored in the `docs/` folder.
 
-Note: `getKey` depends on store support. It works with the MemoryStore, but may
-not work with other stores. Calling it will throw an error if the store does not
-have a `get` method.
+> Thanks to
+> [Mintlify](https://mintlify.com/?utm_campaign=devmark&utm_medium=readme&utm_source=express-rate-limit)
+> for generously hosting the documentation!
```

Comment:
After rebasing, we can take the text and logo Arnie added and move it up to here. (And probably just drop the header that I added, since I think it kind of makes sense in this section.)